lib/sites-base.js

```diff
@@ -549,5 +555,41 @@ module.exports = {
       });
     });
 
+    self.apos.migrations.add('normalize-case', async () => {
+      return self.apos.migrations.eachDoc({ type: 'site' }, async doc => {
+        const $set = {};
+        if ((typeof doc.shortName === 'string') && (doc.shortName !== doc.shortName.toLowerCase())) {
+          const happy = await self.apos.docs.db.findOne({
```

Comment:
Added conflict prevention. If an existing site object has the right name (lowercase), it should win, because it actually works. A site object with a mixed case name already doesn't work, so it's no big deal if we have to add an id to its shortname for now to prevent a conflict.

Reply:
(In the field there is a strong likelihood that if any site objects have mixed case shortnames, they are hanging out in the trash at this point.)